src/infra/redis/ioredis.adapter.ts

```diff
@@ -0,0 +1,197 @@
+import { Redis } from 'ioredis';
+import { Logger } from '../logger/index.js';
```

Comment:
import { Logger } from '../logger'; ?

Reply:
Das funktioniert hier in dem Projekt leider nicht mehr. Daher ist 
```typescript
import { Logger } from '../logger/index.js';
``` 
die korrekte Schreibweise.

Reply:
Das muss mir irgendwann noch jemand erklären. :)